src/main/resources/assets/computercraft/lang/de_DE.lang

```diff
@@ -0,0 +1,64 @@
+tile.computercraft:computer.name=Computer
+tile.computercraft:advanced_computer.name=Erweitertetr Computer
+tile.computercraft:drive.name=Diskettenlaufwerk
+tile.computercraft:printer.name=Drucker
+tile.computercraft:monitor.name=Monitor
+tile.computercraft:advanced_monitor.name=Erweiterter Monitor
+tile.computercraft:wireless_modem.name=WLAN-Modem
+tile.computercraft:wired_modem.name=LAN-Modem
+tile.computercraft:cable.name=LAN-Kabel
+tile.computercraft:command_computer.name=Befehlscomputer
+tile.computercraft:advanced_modem.name=Ender-Modem
+tile.computercraft:speaker.name=Lautsprecher
+
+tile.computercraft:turtle.name=Roboter
```

Comment:
Not sure what to think about this. Wouldn't it be better to keep the term "Turtle" since it's more specific?